cluster/images/hyperkube/Dockerfile

```diff
@@ -14,5 +14,8 @@
 
 FROM BASEIMAGE
 
-# Copy the hyperkube binary
+# Cleanup all the soft links
+ADD binaries.tgz /usr/local/bin
```

Comment:
how does this line clean up soft links?

Reply:
@liggitt by replacing the files :)

Reply:
once this gets in, i will follow up to remove stuff from the base image (and this should continue to work even after that)

Reply:
> If `<src>` is a local tar archive in a recognized compression format (identity, gzip, bzip2 or xz) then it is unpacked as a directory.

ah

Reply:
`ADD` is `COPY`'s magical friend :D